core/lib/spree/core/engine.rb

```diff
@@ -64,6 +64,15 @@ class Engine < ::Rails::Engine
             caller
           )
         end
+
+        if Spree::Config.run_order_validations_on_order_updater == false
```

Comment:
Any reason for not allowing also other falsy values (ie. `nil`) here?

Reply:
I don't think it can be `nil` or any other thing since it's initialized with `true` as default. Of course, unless you initialize this with a non-boolean value. Anyway, the only accepted value that should not raise a deprecation error is `true` so I'm going to change this, thanks!